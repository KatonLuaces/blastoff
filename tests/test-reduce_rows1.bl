A = [1,2;
    3,4;
    5,6];
B = +%A;
C = *%A;
print(toString(B));
print(toString(C));