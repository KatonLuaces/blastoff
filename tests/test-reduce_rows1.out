3
7
11
2
12
30